lib/services/morea_firestore.dart

```diff
@@ -206,16 +265,15 @@ class MoreaFirebase extends BaseMoreaFirebase {
   Future<void> uploaddevtocken(
       var messagingGroups, String token, String userUID) async {
     Map<String, dynamic> tokendata = {'devtoken': token};
-    if(messagingGroups != null)
-    for (var u in messagingGroups.keys) {
-      if (messagingGroups[u]) {
-        await crud0.setData('groups/$u/Devices', userUID, tokendata);
+    if (messagingGroups != null)
+      for (var u in messagingGroups.keys) {
```

Comment:
Ist jetzt messagingGroups im Midata group Format? also eine Nr. z.B. 4013 oder Drason (Buebe)? falls zweitens hier die Methode: `convWebflowtoMiData(String stufe)` einfügen

Reply:
messaging Groups ist noch eine Map<String, bool> mit String = (z. B) Drason (Buebe)

Reply:
Ich meine wegen der Programmzeile unten
`await crud0.setData('groups/$u/Devices', userUID, tokendata)`
Wenn ich das richtig verstehe wird der Devicetoken mit Weblfow name gespeichert
![grafik](https://user-images.githubusercontent.com/32130052/71081019-9fe6d800-218e-11ea-882a-e8313e49c85d.png)

Reply:
Also soweit ich weiss wird der Token unter der Midata Nummer gespeichert, weil als ich die MessagePage angepasst habe, ich die Senderliste fürs verschicken der Nachrichten aus dem Dokument mit der Midata Gruppennummer nehme und nicht aus dem Webflow Namen.
Aber sicher bin ich nicht. Auf jeden Fall habe ich nichts an dem geändert verglichen zum Masterbranch. In diesem Branch habe ich nur am WerChunnt gearbeitet.